samples/samples/snippets.py

```diff
@@ -2862,6 +2903,9 @@ def insert_customers(transaction):
             )
         )
 
+    instance = spanner_client.instance(instance_id)
```

Comment:
where are these handwritten objects used? If not needed, can we perform a search in the entire folder and remove the references of handwritten objects?

Reply:
We need this to do data client inserts.